metaflow/task.py

```diff
@@ -275,24 +275,63 @@ def clone_only(self, step_name, run_id, task_id, clone_origin_task, retry_count)
         )
         metadata_tags = ["attempt_id:{0}".format(retry_count)]
         output.clone(origin)
+        # Get original metadata and extract fields we care about
+        metadata_fields = [
+            "conda_env_id",
```

Comment:
Why do we need `conda_env_id`? We are also leaking information about a plugin here.